photoneo_module:
  plugin_name: pho/virtual_robot_plugin
  vision_controller_ip: 192.168.1.6   #photoneo_server_address
  binpicking_port: 11004               #photoneo_server_port
  robot_controller_ip: 192.168.1.5     #state_listener_address
  state_server_port: 11005             #state_server_port
  pc_server: true
  num_of_joints: 6

virtual_robot:
  cycle_time: 30
  moving_robot_time: 2
  start_pose: [0, 0.2, 0.2, 0, 0, 0 ]
  end_pose: [0, 0.2, 0.2, 0, 0, 0 ]
